### DISCUSSION TITLE
Azure Active Directory authentication in .NET MAUI

### DISCUSSION THREAD

**Discussion by giscus** [VladislavAntonyuk/vladislavantonyuk.github.io#62] [Announcements]
# Azure Active Directory authentication in .NET MAUI

Create a small mobile application, which requires user authentication, using .NET MAUI (AD and AD B2C).

https://vladislavantonyuk.github.io/articles/Azure-Active-Directory-authentication-in-.NET-MAUI/

**Comment by siamaksh1367** [comment]
private async Task OnLoginClicked()
{
    try
    {
        var result = await authService.SignInInteractively(CancellationToken.None);
        await GetResult(result);
    }
    catch (MsalClientException ex)
    {
        await Console.Out.WriteLineAsync(ex.ErrorCode);
    }
}
Microsoft.Identity.Client.MsalServiceException: 'Non-HTTPS URL redirect is not supported in a web view. This error happens when the authorization flow, which collects user credentials, gets redirected to a page that is not supported, for example if the redirect occurs over http. This error does not trigger for the final redirect, which can be http://localhost, but for intermediary redirects.Mitigation: This usually happens when using a federated directory which is not setup correctly. '

I keep getting this error on this line . any idea how to fix it ?
